src/services/mongoose/phone-code.ts

```diff
@@ -0,0 +1,38 @@
+import { VALIDITY_TIME_CODE } from "@config/app"
+import {
+  CouldNotFindPhoneCodeError,
+  RepositoryError,
+  UnknownRepositoryError,
+} from "@domain/errors"
+import moment from "moment"
+import { PhoneCode } from "./schema"
+
+export const PhoneCodesRepository = (): IPhoneCodesRepository => {
+  const findRecent = async ({
+    phone,
+    code,
+  }: {
+    phone: PhoneNumber
+    code: PhoneCode
+  }): Promise<true | RepositoryError> => {
+    try {
+      const phoneCode = await PhoneCode.findOne({
+        phone,
+        code,
+        created_at: {
+          $gte: moment().subtract(VALIDITY_TIME_CODE, "seconds"),
```

Comment:
please use only date (as far as I remember we are trying to remove moment package)

Reply:
just a copy paste here. I recommend doing this in a new PR.